### PR TITLE
fix issue 14933

### PR DESCRIPTION
**Pull request by BBasile**

### CODE REVIEW

File: pragma.dd [hunk 1]
```diff
@@ -124,8 +124,9 @@ pragma(lib, "foo.lib");
 
 
     $(DT $(LNAME2 mangle, $(D mangle)))
-    $(DD Overrides the default mangling for a symbol. This allows
-        linking to a symbol which is a D keyword, which would normally
+    $(DD Overrides the default mangling for a symbol. In DMD, it's only effective
```
Comment (9rnsr) [inline pragma.dd]: `In DMD` is not necessary.

**Comment by BBasile** [comment]
Unfortunately the PR was pushed before your explanation on the bug tracker, after what it was clear that neither GDC or LDC can implement it differently. Anyway it's ok now.

**Comment by 9rnsr** [comment]
@BBasile Thanks for your contribution!

**Comment by 9rnsr** [comment]
Link to the bugzilla ticket: https://issues.dlang.org/show_bug.cgi?id=14933

**Pull request merged**